js/about/newtab.js

```diff
@@ -257,6 +264,105 @@ class NewTabPage extends React.Component {
     return name.charAt(0).toUpperCase()
   }
 
+  launchBraveCore () {
+    const braveCoreInstallPath = this.state.versionInformation && this.state.versionInformation.getIn(['initState', 'braveCoreInstallPath'])
+    if (braveCoreInstallPath) {
+      aboutActions.launchBraveCore()
+    }
+  }
+
+  openHelp () {
+    window.location = 'https://support.brave.com/hc/en-us/articles/360018538092'
```

Comment:
i think it is better UX here to use the newTabRequested action https://github.com/brave/browser-laptop/pull/15309#discussion_r238865331